libraries/cmake/formula/modules/utils.cmake

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) 2014-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed in accordance with the terms specified in
+# the LICENSE file found in the root directory of this source tree.
+
+cmake_minimum_required(VERSION 3.13.3)
+include(ExternalProject)
+
+set(OSQUERY_FORMULA_INSTALL_DIRECTORY "${CMAKE_BINARY_DIR}/installed_formulas")
+
+function(importFormula library_name)
```

Comment:
Only the first configure step works due to a bug, we should fix this!

Reply:
This has been fixed; we will still have to port the original formula settings (for Windows and macOS) after the first release